code/modules/modular_computers/file_system/programs/arcade.dm

```diff
@@ -0,0 +1,166 @@
+/datum/computer_file/program/arcade
+	filename = "arcade"
+	filedesc = "Nanotransen Micro Arcade"
+	ui_header = "smmon_0.gif"
+	program_icon_state = "arcade"
+	extended_desc = "This port of the classic game 'Outbomb Cuban Pete', redesigned to run on tablets, with thrilling graphics and chilling storytelling."
+	requires_ntnet = FALSE
+	network_destination = "arcade network"
+	size = 6
+	tgui_id = "ntos_arcade"
+	ui_x = 600
+	ui_y = 400
+
+	var/game_active = TRUE //Checks to see if a game is in progress.
+	var/pause_state = FALSE //This disables buttons in order to prevent multiple actions before the opponent's actions.
+	var/boss_hp = 45
+	var/boss_mp = 15
+	var/player_hp = 30
+	var/player_mp = 10
+	var/ticket_count = 0
+	var/heads_up = "Nanotrasen says, winners make us money."//Shows the active display text for the app
+
+/datum/computer_file/program/arcade/proc/game_check(mob/user)
+	sleep(5)
+	if(boss_hp <= 0)
+		heads_up = "You have crushed the forces of evil! Rejoyce!"
```

Comment:
Rejoyce?

Reply:
It's not one of my PRs without at LEAST one spelling error.